apps/src/templates/progress/ProgressDetailToggle.story.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ProgressDetailToggle from './ProgressDetailToggle';
+import { Provider } from 'react-redux';
+import {combineReducers, createStore} from 'redux';
+import progress, { setIsSummaryView } from '@cdo/apps/code-studio/progressRedux';
+
+export default storybook => {
+  const storyTable = [];
+
+  // Stick these in anonymous functions so that each instance gets its own
+  // store
+  (function () {
+    const store = createStore(combineReducers({progress}));
```

Comment:
Storybook+redux! :+1:

A thought - I wonder if this would read nicer if each story was a named helper instead of an IIFE.  It might be a first step toward deduplicating store creation and the provider wrapper.

```javascript
function isSummaryIsTrue() {
  const store = createStore(combineReducers({progress}));
  return {
    name: 'isSummary is true',
    story: () => (
      <Provider store={store}>
        <ProgressDetailToggle/>
      </Provider>
    )
  };
}

//...

storybook
  .storiesOf('ProgressDetailToggle', module)
  .addStoryTable([
    isSummaryIsTrue(),
    isSummaryIsFalse()
  ]);
```